content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/lifo.rst

```diff
@@ -0,0 +1,66 @@
+============
+LIFO removal
+============
+
+The *Last In, First Out* (LIFO) removal strategy picks the **newest** products on-hand, based on the
+date they entered a warehouse's stock.
+
+Every time an order is placed for products with the :abbr:`LIFO (Last In, First Out)` method, a
+transfer is created for the lot/serial number that has most recently entered the stock (the **last**
+lot/serial number that entered the warehouse's inventory).
+
+.. warning::
+   In many countries, the :abbr:`LIFO (Last In, First Out)` removal strategy is banned, since it can
+   potentially result in old, expired, or obsolete products being delivered to customers.
+
+Consider the following example, with the product, `Cinder Block`, which is tracked :guilabel:`By
+Lots` in the :guilabel:`Inventory` tab of the product form. Its product category's :guilabel:`Force
+Removal Strategy` is set to :guilabel:`Last In, First Out (LIFO)`.
+
+.. seealso::
+   - :ref:`Set up force removal strategy <inventory/warehouses_storage/removal-config>`
+   - :ref:`Enable lots tracking <inventory/warehouses_storage/lots-setup>`
+   - :ref:`Check arrival date <inventory/warehouses_storage/arrival_date>`
+   - :doc:`About removal strategies <removal>`
+
+The following table represents the cinder blocks in stock, and their various lot number details.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * -
+     - LOT1
+     - LOT2
+     - LOT3
+   * - On-hand stock
+     - 10
+     - 10
+     - 10
+   * - :ref:`Created on <inventory/warehouses_storage/arrival_date>`
+     - June 1
+     - June 3
+     - June 6
+
+To see the removal strategy in action, create a :ref:`delivery order <inventory/delivery/one-step>`
+for seven cinder blocks, either by going to the :menuselection:`Sales app` and creating a new
+quotation, or from the delivery orders dashboard in :menuselection:`Inventory app --> Operations -->
+Deliveries`.
+
+:guilabel:`Confirm` the sales order, or click :guilabel:`Mark as Todo` on the draft delivery order.
+
+Doing so creates the delivery order, and the newest lot numbers are reserved, according to the
+:abbr:`LIFO (Last In, First Out)` removal strategy.
```

Comment:
I think you should remove the instructions for creating a delivery order directly. It gets a bit confusing when you're trying to include instructions for both that and creating a delivery order from a sales order.